include/n2n_typedefs.h

```diff
@@ -253,7 +253,15 @@ typedef enum n2n_pc
    n2n_query_peer=11           /* ask supernode for info on a peer */
   } n2n_pc_t;
 
-#define N2N_FLAGS_OPTIONS               0x0080
+
+/* This value is used in sn_utils.c process_udp, flags large than this value will make supernode treat packet's header as encrypted.
+ * If add new flags, must change N2N_FLAGS_MAX_VALUE at the same time.*/
+#define N2N_FLAGS_MAX_VALUE            0x0100
+
+/* Set at edge if the packet paylod is ARP. Supernode always forward ARP packet.*/
+#define N2N_FLAGS_ARP_PACKET            0x0100
+
+#define N2N_FLAGS_OPTIONS               0x0080 // TODO: this flags not need any more, can be delete.
```

Comment:
As far as I see it, `N2N_FLAGS_OPTIONS` is meant to indicate the first available value for (new) user options, so `0x0080` could be used as `ARP` while `N2N_FLAG_OPTIONS` could be shifted to `0x0100` (and optionally rename it to `N2N_FLAG_MAX_VALUE` if more convenient)